### PR TITLE
Team bios

### PR DESCRIPTION
**Pull request by ElenaFdR**
This PR includes:
- link from homepage
- index page for bios (includes text, photos for me and Ben to test the layout)
- pages for each person's specific bio (created but not populated)

In the interest of getting a bios page up promptly, this PR:
- uses the default page layout on the bios index page, rather than a bespoke layout
- relies on people to provide their own photos, rather than coordinating matching photos

-----
[View rendered staff/ben-bio.md](https://github.com/publiccodenet/http/blob/Our-team/staff/ben-bio.md)
[View rendered staff/boris-bio.md](https://github.com/publiccodenet/http/blob/Our-team/staff/boris-bio.md)
[View rendered staff/claus-bio.md](https://github.com/publiccodenet/http/blob/Our-team/staff/claus-bio.md)
[View rendered staff/elena-bio.md](https://github.com/publiccodenet/http/blob/Our-team/staff/elena-bio.md)
[View rendered staff/eric-bio.md](https://github.com/publiccodenet/http/blob/Our-team/staff/eric-bio.md)
[View rendered staff/index.md](https://github.com/publiccodenet/http/blob/Our-team/staff/index.md)
[View rendered staff/jan-bio.md](https://github.com/publiccodenet/http/blob/Our-team/staff/jan-bio.md)
[View rendered staff/mirjam-bio.md](https://github.com/publiccodenet/http/blob/Our-team/staff/mirjam-bio.md)

### CODE REVIEW

**Comment by ElenaFdR** [comment]
This is ready to go, barring some weirdness in the local rendering (which @ericherman thinks might be due to my gemfiles, rather than this content - but do check as part of the approvals process!)

File: staff/elena-bio.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (bvhme) [inline staff/elena-bio.md]: Unnecessary whitespace

File: staff/eric-bio.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Eric Herman is the Foundation for Public Code's Lead Codebase Steward for Quality. With two decades of experience, he brings the perspectives of codebases of many languages and development practices.
```
Comment (bvhme) [inline staff/eric-bio.md]: Missing a H1

File: staff/eric-bio.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+He worked as a developer for MySQL and remains active in the Free and Open Source Software database community. He continues to contribute to MySQL and MariaDB. Eric serves as the chairman of the board of directors of the MariaDB Foundation.
+
+Formerly a Principal Developer at Booking<span></span>.com, he has worked on the data infrastructure of one of the top 10 largest ecommerce sites. During his time at Booking<span></span>.com, he developed performance enhancements to the Perl programming language which were contributed upstream for all users of Perl to benefit from.
```
Comment (bvhme) [inline staff/eric-bio.md]: Remove the HTML `<span></span>` from this sentence.

Comment (ericherman) [reply staff/eric-bio.md]: Removing the span will link the employer ... twice ... which is not what we want. Do you have an alternative way to write it such that it is not linked?

Comment (ericherman) [reply staff/eric-bio.md]: addressed with commit caf364a09d38e3d12abe7c779be312e7bb9a39ac

>    Remove "example<span></span>.com" avoid-link-trick
>    
>    With Jekyll, the "span trick" is not needed to avoid the link.
>    It does not matter if the repo view version links, we only care about
>    the html rendered output.

File: staff/index.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (bvhme) [inline staff/index.md]: Unnecessary whitespace
